grunt-tasks/purifycss.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function(grunt) {
+  grunt.config("purifycss", {
+    options: {
+      rejected: true,
+      info: true
+    },
+    target: {
+      src: [
+        "webcompat/templates/*.html",
+        "webcompat/templates/**/*.html",
```

Comment:
`**` matches zero or more directories (when its the only thing in the path), so the first line is redundant. Just `"webcompat/templates/**/*.html"` should be sufficient.